tidy3d/plugins/smatrix/component_modelers/terminal.py

```diff
@@ -193,16 +194,25 @@ def port_voltage(port: LumpedPort, sim_data: SimulationData) -> xr.DataArray:
             e_component = "xyz"[port.voltage_axis]
             field_data = sim_data[f"{port.name}_E{e_component}"]
             e_field = field_data.field_components[f"E{e_component}"]
+
+            # Remove E field outside the port region
+            e_field = e_field.sel(
+                {
+                    e_component: slice(
+                        port.bounds[0][port.voltage_axis], port.bounds[1][port.voltage_axis]
+                    )
```

Comment:
When you use `slice` here, is it exclusive on both start and end? My impression is that it is inclusive in the beginning.

Reply:
Now I understand your point, I added which makes the excludes the boundaries. I didn't see a better way so I used `drop_sel`